requirements.txt

```diff
@@ -0,0 +1 @@
+trackingprotection_tools==0.4.6
```

Comment:
According to some good practice notes on [Firefox source docs here](https://firefox-source-docs.mozilla.org/python/index.html#using-a-python-package-index), I suggest pinning the requirements. [Here](https://stackoverflow.com/questions/34645821/pip-constraints-files/36848206#36848206) is a suggestion on how to do this.

Reply:
@skim1102 I just want to clarify you're asking for the requirement to be hashed, for example using the 'hashin' library. On doing so, Travis fails because "all requirements must have their versions pinned with ==". How do I go about pinning the hashed dependencies? 

On a seperate note, I added trackinprotection_tools to requirements.txt exactly how it exists in requirements.txt for shavar-list-creation. Does that requirements.txt file need to be amended too?

Reply:
Hi @skim1102, Just wondered if you had any feedback on this?